### DISCUSSION TITLE
PHP POSTGRES project into flutter with serverpod convertion with the db data any possibilities

### DISCUSSION THREAD

**Discussion by JAYAMURUGANJ** [serverpod/serverpod#2458] [Q&A] [answered]
a project developed by php with postgres. i like to chage that into fully in flutter with the help of serverpod

**Comment by vlidholt** [comment, marked as answer]
Assuming you have read up on the basics of Serverpod, you will need to:

1. Make sure all tables have an `id` column as its primary key.
2. Create a model (yaml) file for each table. The model should not include the `id` field as it's added automatically. Set the `table` key to the name of your database table. Make sure there is a field for each column that uses the right type.
3. If there are tables with columns that doesn't map to a Serverpod type, you can exclude these tables and manage them manually. You can opt out by setting the `managedMigration` key to `false`.
4. Make sure to run `serverpod generate` to build all the models in Dart. Then, you can start the server and you will get warnings if there is anything that is out of sync.

Hope this helps! 🙂
